src/components/views/rooms/EventTile.js

```diff
@@ -564,7 +569,8 @@ module.exports = WithMatrixClient(React.createClass({
 }));
 
 module.exports.haveTileForEvent = function(e) {
-    if (e.isRedacted()) return false;
+    // Only messages have a tile (black-rectangle) if redacted
+    if (e.isRedacted() && e.getType() !== 'm.room.message') return false;
```

Comment:
Conclusion is to do a separate PR for other event types (where we probably just skip rendering the specific tile altogether): https://matrix.to/#/!DdJkzRliezrwpNebLk:matrix.org/$148905194216eubJF:ldbco.de